chipsec/modules/tools/secureboot/te.py

```diff
@@ -465,19 +473,20 @@ def confirm():
     logger().warn("* - On Windows, use recovery mode which should automatically restore correct executables")
     logger().warn("*")
     logger().warn("***************************************************************************************")
-    s = cs_input( "Type 'yes' to continue running the tool > " )
-    if s != 'yes': sys.exit( 0 )
+    s = input("Type 'yes' to continue running the tool > ")
+    if s != 'yes':
```

Comment:
We've moved some of these checks to `if s.lower() not in ['yes','y']:` (or equivalent) as kind of a quality of life improvement.  But this works and is current behavior.